code/__DEFINES/access.dm

```diff
@@ -70,6 +70,8 @@
 #define ACCESS_FL 73 //NSV13 - FL tagged decorated fighter
 #define ACCESS_MINING_ENGINEERING 74
 #define ACCESS_MINING_BRIDGE 75
+#define ACCESS_ENTER_GENPOP 76 //NSV13 - Access to enter genpop
```

Comment:
Can you not use one-way door access helpers here? like in medbay

Reply:
Dunno if the one-way door helper will work here. Since the turnstill technically is not an airlock.
I'll give this some testing though once I got the turnstills working.